lib/components/Stepper/style.scss

```diff
@@ -0,0 +1,63 @@
+@import '../../theme/tokens/facade';
+
+@mixin isDisabledState() {
+	&[disabled],
+	&[aria-disabled='true'] {
+		cursor: not-allowed;
+
+		&:not(.loading) {
+			@content;
+		}
+	}
+}
+
+$handleSize: 24px;
+
+.root,
+.handle {
+	flex-direction: row;
+	display: flex;
+	align-content: center;
+	justify-content: center;
+	align-items: center;
+}
+
+.root {
+	font-size: var(--global--fontSize--5);
+	line-height: var(--global--fontSize--5);
+	color: getColour(gray, 900);
+	border-radius: 4px;
+	border: solid 1px getColour(gray, 300);
+	box-shadow: 0 1px 10px 0 rgba(0, 0, 0, 0.03),
+		0 4px 5px 0 rgba(0, 0, 0, 0.03), 0 2px 4px -1px rgba(0, 0, 0, 0.05);
+	padding: calc(var(--global--space--3) - 1px)
+		calc(var(--global--space--2) - 1px);
+
+	@include isDisabledState() {
+		cursor: not-allowed;
+	}
+}
+
+.handle {
+	width: $handleSize;
+	height: $handleSize;
+	background-color: getColour(green, 700);
+	border-radius: 50%;
+	cursor: pointer;
+	-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
Should this be more of a `_reset.scss` concern (see lib/theme/base/_reset.scss, and lib/theme/base/_box-sizing.scss), where we already do a `*, *:before, *:after {}`, and put it in there. And it's more a case of opt-in, to change this tap colour to something, if we need too?